backend/src/main/java/com/speakfluid/backend/service/TranscriptService.java

```diff
@@ -15,6 +15,14 @@
 
 import java.io.IOException;
 
+/**
+ * TranscriptService is used to query MongoDB.
+ *
+ * @author  Kai Zhuang
+ * @version 1.0
+ * @since   2022-11-12
+ */
+
 @Service
 public class TranscriptService {
```

Comment:
Is there a better name for this? I can't rmr if they call this type of class a Gateway or something